code/LINDA/LINDA_turf_tile.dm

```diff
@@ -270,14 +270,10 @@ turf/simulated/proc/share_temperature_mutual_solid(turf/simulated/sharer, conduc
 
 
 atom/movable/var/pressure_resistance = 5
-atom/movable/var/last_forced_movement = 0
 
 atom/movable/proc/experience_pressure_difference(pressure_difference, direction)
-	if(last_forced_movement >= air_master.current_cycle)
```

Comment:
The problem is that, if you remove this sanity check, whether something gets moved multiple times in a single tick depends entirely on the order LINDA processes turfs. If a pressure check on one turf moves something onto a turf that hasn't been processed yet, it can get moved again when its new turf processes. Under the right conditions, it may be possible for something to get pulled from one end of a room, all the way across it, and into space, all in a single tick. There's no way to know for sure without knowing that LINDA does its own sanity checking.

> I'd like you to test with the game actually, well, running.

I tested with the game running, and LINDA moved stuff fine when I overpressurized an area, but a depressurizing area (as you'd get from hull breaches) only moved things on the turfs adjacent to space, regardless of how quickly air on other turfs was actually moving. You can't really test whether pressure differences are moving stuff correctly when LINDA doesn't appear to be properly processing them in the first place.